common/activities/activity_types/tournament.txt

```diff
@@ -616,13 +621,19 @@
 		}
 		if = {
 			limit = {
-				#OR = {
+				OR = {
 					has_building_or_higher = generic_university
+					has_building_or_higher = gearshaft_university
+					has_building_or_higher = scholomance
+					has_building_or_higher = falthrien_academy
+					has_building_or_higher = narthalas_academy_02
+					has_building_or_higher = karazhan_01
```

Comment:
I can't see what Kharazan is doing here.
It's certainly a place where some people can study, but it's not as crowded as other academies/universities. It remains quite lonely. Ok Medivh makes failures, but does it have anything to do with the building?